context-free.md

```diff
@@ -1,21 +1,24 @@
 # Context Free
 
-##  Who is this programming language for?
-
+## Who is this programming language for?
+for people who want to generate, instead of draw, certain kind of shape through computer.
 
 ## What is easy to do in this language? Why is it easy?
-
+It is easy to generate shapes people want because we do not need to know much function to do that. And the parameters are easy to learn and understand. In addition, the code in general is easy to read and understand. So even you are not familiar with coding, you can still do that.
```

Comment:
I also think that this program is pretty straightforward and relatively quick to learn